### PR TITLE
W-12482132: Add EOL banner to Flow Designer articles

### PR DESCRIPTION
**Pull request by fermujica**
Considering adding a page attribute to display the support status banner to the Flow Designer articles

See UI bundle change: https://github.com/mulesoft/docs-site-ui/pull/306

Preview of the banner:
Screenshot 2023-02-13 at 15 55 46

### CODE REVIEW

File: modules/ROOT/pages/about-data-types.adoc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 = Define and Use Data Types in a Flow (Flow Designer)
+:page-support-status: eolReached
```
Comment (hannanelson) [inline modules/ROOT/pages/about-data-types.adoc:2]: Should this be `eolScheduled`? The EOL dashboard says Flow Designer reaches End of Standard Support on 2/15/2024.

Comment (fermujica) [reply modules/ROOT/pages/about-data-types.adoc:2]: I was checking that with the PM. Yes, I thought that the EOL banner was "scheduled".

Comment (hannanelson) [reply modules/ROOT/pages/about-data-types.adoc:2]: Specifically `eolScheduled`. Gary updated the wiki:
https://confluence.internal.salesforce.com/display/MTDT/Using+Banners#UsingBanners-DocsThatAreScheduledtoReachEnd-Of-Life(EOL)

Comment (fermujica) [reply modules/ROOT/pages/about-data-types.adoc]: Addressed: https://github.com/mulesoft/docs-design-center/pull/206/commits/9ead98c8ce603f763b4a2cf3cf871ec557c4b827

**Review by hannanelson — APPROVED** [review]

**Comment by fermujica** [comment]
@dhirsch1 This wasn't supposed to be merged. Why was it merged yesterday?

**Pull request merged**